src/EFCore.PG/Metadata/NpgsqlRelationalAnnotations.cs

```diff
@@ -0,0 +1,26 @@
+using System.Collections.Generic;
+using System.Linq;
+using JetBrains.Annotations;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Metadata;
+
+namespace Npgsql.EntityFrameworkCore.PostgreSQL.Metadata
+{
+    /// <summary>
+    /// Provides relational-specific annotations specific to Npgsql.
+    /// </summary>
+    public class NpgsqlRelationalAnnotations : RelationalAnnotations, INpgsqlRelationalAnnotations
```

Comment:
Same as for INpgsqlRelationalAnnotations - these seem to belong in NpgsqlModelAnnotations...